app/src/main/java/io/homeassistant/companion/android/sensors/PowerSensorManager.kt

```diff
@@ -99,7 +99,7 @@ class PowerSensorManager : SensorManager {
             dozeState,
             icon,
             mapOf(
-                "ignoring_battery_optizimations" to powerManager.isIgnoringBatteryOptimizations(packageName)
+                "ignoring_battery_optimizations" to powerManager.isIgnoringBatteryOptimizations(packageName)

```

Comment:
This is a breaking change and unrelated to this PR, PRs should only contain changes that are part of it.

Reply:
I sort of regarded it as part of it - I had to take a crash course in sensor writing, and as such used the PowerManager as an example, so got to read the code for that in quite some depth, thus spotting this problem.

I also assumed this was just an internal name - I couldn't find any use of that string elsewhere in the app, so didn't think it broke anything.  Happy to drop it from the PR if that's how things are done here.

Reply:
This is an attribute of the sensor and changing it has the potential to break any template depending on it, we like to give a heads up on breaking changes to users and not just make a undocumented change. So a separate PR highlighting the breaking change can minimize its impact.

Reply:
@dshokouhi Looking for advice as to how this project expects me to do this - should I make a further commit that reverts this line, or do I rewrite history, and simply force push the existing commit without this line changed?

Reply:
@zmarties its ok to submit another commit reverting the change :)